src/webgpu/api/operation/buffers/map_oom.spec.ts

```diff
@@ -72,14 +72,11 @@ g.test('mapAsync')
     }
   });
 
-g.test('mappedAtCreation,full_getMappedRange')
+g.test('mappedAtCreation')
   .desc(
-    `Test creating a very large buffer mappedAtCreation buffer should produce
-an out-of-memory error if allocation fails.
-  - Because the buffer can be immediately mapped, getMappedRange throws an OperationError only
-    because such a large ArrayBuffer cannot be created.
-  - unmap() should not throw.
-  `
+    `Test creating a very large buffer mappedAtCreation buffer should throw a RangeError only
+     because such a large ArrayBuffer cannot be created when we initialize an active buffer mapping.
```

Comment:
nit: an ArrayBuffer isn't actually created until getMappedRange
```suggestion
     because such a large allocation cannot be created when we initialize an active buffer mapping.
```